src/core/eme/check_key_statuses.ts

```diff
@@ -49,12 +50,14 @@ export default function checkKeyStatuses(
     /* tslint:enable no-unsafe-any */
     // Hack present because the order of the arguments has changed in spec
     // and is not the same between some versions of Edge and Chrome.
-    const [keyStatus, keyId] = (() => {
+    const [keyStatus, keyStatusKeyId] = (() => {
       return (typeof _arg1  === "string" ? [_arg1, _arg2] :
                                            [_arg2, _arg1]
              ) as [MediaKeyStatus, ArrayBuffer];
     })();
 
+    const keyId = getUUIDKidFromKeyStatusKID(keySystem.type,
```

Comment:
Considering that `keySystem.type` is just the `keySytems[].type` argument from the API, which could be either:
  1. the full reverse domain name identifying the wanted CDM (e.g. `"com.microsoft.playready"`) or
  2. just an "alias" defined in the config (e.g. just `"playready"`)

I fear that we could be having future false negatives here without knowing it by creating another alias for a playready key system in the config which doesn't contain the "playready" string in it (a credible example would be to name one with the name of the platform instead, like `"nintendo-64"`).

To fix that, we could still rely on `keySystem.type` but generalize an utils function to translate it to its real name or we could directly be communicating the "real" key system type to this function.

Reply:
I see what you mean.